specification/hybridcloud/resource-manager/Microsoft.HybridCloud/preview/2023-01-01-preview/hybridCloud.json

```diff
@@ -0,0 +1,1002 @@
+          "items": {
+            "$ref": "#/definitions/CloudConnection"
+          },
+          "description": "A list of cloud connection resources."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to hold the next page of results"
+        }
+      },
+      "description": "Response for the ListCloudConnections API service call"
+    },
+    "CloudConnectionPropertiesFormat": {
+      "type": "object",
+      "properties": {
+        "cloudConnector": {
+          "$ref": "#/definitions/ResourceReference",
+          "description": "The cloud connector which discovered the remote resource.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "remoteResourceId": {
+          "type": "string",
+          "description": "Identifier for the remote cloud resource",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "virtualHub": {
+          "$ref": "#/definitions/ResourceReference",
+          "description": "The virtualHub to which the cloud connection belongs.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "sharedKey": {
+          "type": "string",
+          "description": "Shared key of the cloud connection.",
+          "x-ms-secret": true
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The provisioning state of the cloud collection resource.",
+          "readOnly": true
+        }
+      },
+      "description": "Cloud connection resource properties."
+    },
+    "CloudConnection": {
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "description": "Resource which represents the managed network connection between Azure Gateways and remote cloud gateways.",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CloudConnectionPropertiesFormat",
+          "description": "Properties of the cloud connection."
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "ResourceFilter": {
+      "type": "object",
+      "description": "Remote Cloud Resource filter.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Resource name to filter."
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type to filter."
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location to filter."
+        },
```

Comment:
There is a Linter Warning about the below:

Property location must have 'x-ms-mutability':['read', 'create'] extension defined.

Reply:
This is not correct from linter. This location actually refers to remote cloud location.

Reply:
In that case... perhaps you should call it something else and not 'location' since it is easily incorrectly assumed to be an Azure location?

Reply:
Changing it to region